Makefile

```
@@ -23,6 +23,10 @@ install: build
 clean:
 	cd $(CURDIR); rm -f karmor
 
+.PHONY: test
+test:
+	cd log; go test
```

```suggestion
	cd $(CURDIR); go test -v ./...
```

This would keep this compatible with addition of future tests.